bitnami/fluentd/templates/tls-certs.yaml

```diff
@@ -16,6 +16,7 @@ type: kubernetes.io/tls
 data:
   tls.crt: {{ .certificate | b64enc }}
   tls.key: {{ .key | b64enc }}
+  rootCA.crt: {{ .caCertificate | b64enc }}
```

Comment:
`tls.caCertificate` is not present in the values.yaml, see https://github.com/bitnami/charts/commit/83ad4ad8664d6a975beca04ed90800a4efd3a40c. It should be removed from the README table and not used here

Reply:
Ok, the tls.caCertificate was removed but how can i configure a custom certificate autority ?
I need to use this type of following configuration but i could not declare CA certicate for the moment : 
```
<transport tls>
            cert_path /opt/bitnami/fluentd/certs/tls.crt
            private_key_path /opt/bitnami/fluentd/certs/tls.key
            ca_path /opt/bitnami/fluentd/certs/rootCA.crt
</transport>
````

Reply:
Yes, it is true, in that case. Can you add the parameter in the values.yaml? In that case the certificate should be mounted as a volume, see other examples like `phpmyadmin/templates/deployment.yaml`

Reply:
In this file, can you add a similar entry to the ones regarding `tls.crt` and `tls.key` at the end of the file?
```diff
 data:
   tls.crt: {{ required "A valid .Values.tls.certificate entry required!" .Values.tls.certificate | b64enc }}
   tls.key: {{ required "A valid .Values.tls.key entry required!" .Values.tls.key | b64enc }}
+  rootCA.crt: {{ required "A valid .Values.tls.caCertificate entry required!" .Values.tls.caCertificate | b64enc }}
```